automation/jleechanorg_pr_automation/openai_automation/oracle_cli.py

```
@@ -58,32 +62,50 @@ def __init__(
         self.timeout = timeout
         self.playwright: Optional[Playwright] = None
         self.browser: Optional[Browser] = None
+        self.context: Optional[BrowserContext] = None
         self.page: Optional[Page] = None
 
+    async def _save_auth_state(self) -> None:
+        if self.context is None:
+            return
+        await self.context.storage_state(path=str(AUTH_STATE_PATH))
+        print(f"💾 Authentication state saved to {AUTH_STATE_PATH}")
```

Ruff (0.14.8)

`print` found

Remove `print`

(T201)